lib/suma/async/reset_code_update_twilio.rb

```diff
@@ -30,6 +30,12 @@ def update_verification(code, status)
         # the 'send via twilio verify' logic in SmsTransport, so we only update twilio when we use that template.
         Suma::Message::SmsTransport.verification_delivery?(md)
     verification_id = Suma::Message::SmsTransport.transport_message_id_to_verification_id(md.transport_message_id)
-    Suma::Twilio.update_verification(verification_id, status:)
+    begin
+      Suma::Twilio.update_verification(verification_id, status:)
+    rescue Twilio::REST::RestError => e
+      # 404 means twilio has approved, expired or invalidated the code already
+      # https://www.twilio.com/docs/verify/api/verification-check#check-a-verification
+      nil if e.code === 404
```

Comment:
You're not re-raising if the code is *not* a 404.
Also can you explain why this is in the job, and not `update_verification`?

Reply:
Good catch. And I overlooked that the `update_verification` method calls the actual client to make the request, thanks I moved it into the method.